stl/inc/vector

```diff
@@ -458,6 +448,22 @@ extern const bool _Asan_vector_should_annotate;
 #endif // _ACTIVATE_VECTOR_ANNOTATION
 
 #ifdef _INSERT_VECTOR_ANNOTATION
+extern "C" {
+void __cdecl __sanitizer_annotate_contiguous_container(
+    const void* _First, const void* _End, const void* _Old_last, const void* _New_last) noexcept;
+extern const bool _Asan_vector_should_annotate;
+}
+
+#if defined(_M_IX86)
+#pragma comment(linker, \
+    "/alternatename:___sanitizer_annotate_contiguous_container=___sanitizer_annotate_contiguous_container_default")
+#pragma comment(linker, "/alternatename:__Asan_vector_should_annotate=__Asan_vector_should_annotate_default")
+#elif defined(_M_X64) || defined(_M_ARM) || defined(_M_ARM64)

```

Comment:
I observe that there may be interesting ARM64EC (or CHPE) interactions with name mangling, but I can't point to specific issues, so no change requested.